meta-balena-common/recipes-support/hostapp-update-hooks/files/60-balena_migrate_storage/forward

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -o errexit
+
+USER_APP_STORAGE="/mnt/data/docker"
+HOST_APP_STORAGE="/mnt/sysroot/inactive/balena"
+
+printf "[INFO] balena_migrate_storage: Migrating userapp storage to overlayfs... \n"
+env BALENA_A2O_STORAGE_ROOT=$USER_APP_STORAGE a2o-migrate -debug -migrate
+
+printf "[INFO] balena_migrate_storage: Migrating hostapp storage to overlayfs... \n"
+env BALENA_A2O_STORAGE_ROOT=$HOST_APP_STORAGE a2o-migrate -debug -migrate
```

Comment:
Not sure I understand this. Shouldn't this be left for the device to decide?
We fetch an aufs image. And then on-device it becomes overlay? feels wrong.